Formula/pkcs11-tools.rb

```diff
@@ -0,0 +1,24 @@
+class Pkcs11Tools < Formula
+  desc "Tools to manage objects on PKCS#11 crypotographic tokens"
+  homepage "https://github.com/Mastercard/pkcs11-tools"
+  url "https://github.com/Mastercard/pkcs11-tools/releases/download/v2.5.0/pkcs11-tools-2.5.0.tar.gz"
+  sha256 "4e2933ba19eef64a4448dfee194083a1db1db5842cd043edb93bbf0a62a63970"
+  license "Apache-2.0"
+
+  depends_on "bison" => :build
+  depends_on "flex" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "openssl@1.1"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    # no tests, because all tools require the installation of a PKCS#11 module (shared library).
```

Comment:
This doesn't seem like an impediment to testing.

Why can't we install a module for the purposes of testing? Or try to run the tools when we don't have any available to check if we get the behaviour that's expected?

Reply:
The PKCS#11 modules are basically drivers for hardware security modules and only work with specific hardware appliances. I could add a dependency to the softhsm formula which also includes such a module and works without hardware. That dependency would, however, only be necessary for the test cases and not for building or running the pkcs11-tools. If this is acceptable, I would add such tests.

Reply:
Do you mean you could make this formula depend on `softhsm`? That's fine, we can always do
```ruby
depends_on "softhsm" => :test
```